nodehid.js

```diff
@@ -31,11 +31,19 @@ function HID() {
     /* Now we have `this._raw` Object from which we need to
         inherit.  So, one solution is to simply copy all
         prototype methods over to `this` and binding them to
-        `this._raw`
+        `this._raw` if we don't already have a wrapper implemented
     */
-    for(var i in binding.HID.prototype)
-        if(i != "close" && i != "read")
-            this[i] = binding.HID.prototype[i].bind(this._raw);
+    for (var i in binding.HID.prototype) {
+		if (!HID.prototype[i]) {
+			this[i] = (function(fn){
```

Comment:
This is effectively the same as the previous code using `.bind()`, except that it allowed me to log arguments being passed. It isn't needed unless we value that logging - which should probably be behind some sort of flag.